invenio_cli/cli.py

```diff
@@ -17,6 +16,7 @@
 
 import click
 import docker
+from pathlib import Path
```

Comment:
minor: `pathlib` is part of the standard library in Python 3. This one is a backport, because setup.py says this module supports Python 2.7, but I thought we were only supporting Python 2 for previously existing module and only supporting Python 3 for new ones? We could get away with placing that import in the 1st party section and not installing the `pathlib` package.

Reply:
You are right, Py 2 support it out for new modules. removing.